draft-fossati-tls-attestation.md

```diff
@@ -939,14 +939,10 @@ confidential computing properties.
 
 ## IoT Device Onboarding
 
-In this example, an IoT is onboarded to a cloud service provider (or to a
-network operator). In this scenario there is typically no a priori
-relationship between the device and the cloud service provider that 
-will remotely manage the device.
-
-In such scenario, the cloud service provider wants to make sure
-that the device runs the correct version of firmware, has not been 
-rooted, is not emulated or cloned.
+In this scenario, an IoT device is connected to a Device Manager, which could be a cloud service provider or a network operator.
```

Comment:
```suggestion
In this scenario, an IoT device is connected to a remote device management entity, which could be a cloud service provider or a network operator.
```

Reply:
"remote device management entity" does not seem to be a standard term. Just curious to know what else it could be other than CSP or network operator?

Reply:
@muhammad-usama-sardar Remote Entity can be any thing that controls the device. Device Management is an industry standard term and we are just referring that entity here

Reply:
> Device Management is an industry standard term 

Can you point me to any such standard which defines this term "Device management"?

Reply:
Also, standard-wise, [OMA DM](https://www.openmobilealliance.org/release/DM/V1_3-20160524-A/OMA-TS-DM_Protocol-V1_3-20160524-A.pdf) and [oneM2M](https://www.onem2m.org/technical/published-specifications) are two quite popular specs.

*